.emdaer/README.emdaer.md

```diff
@@ -44,7 +44,7 @@ To start with, let's just look at the simplest example:
 const { Handler } = require('lambda-patterns');
 
 module.exports = {
-  yourHandler: Handler.create(event => ({
+  yourHandler: Handler.create({ event } => ({
```

Comment:
Does this work? Looks like it needs parens

Reply:
good catch!